plugins/foreman_virt_who_configure/0.1/index.md

```diff
@@ -0,0 +1,173 @@
+ 
+## 4.1 Creating
+
+In order to create new virt-who configuration, navigate to Insrastructure -> Virt-who configurations and click Create Config button. This takes you to form which guides you through what we need to know in order to successfully configure virt-who. Most of fields provide some inline help, please click the help icon for more information.
+
+Note that every virt-who configuration must exist in an organization. If you haven't selected any organization, you'll see Owner field in the form where you need to choose an Organization. If you already are in some organization context, the Owner field is not displayed and the virt-who configuration will be automatically created in this Organization scope.
+  
+Default values which are prefilled are recommended values which are usually not needed to be changed. This applies to especially to Interval, Hypervisor ID and Satellite server FQDN. The Satellite server FQDN is where the virt-who instance will send reports to so make sure this FQDN is reachable from the host, where the virt-who instance will run. The default value is determined based on what Setting foreman_url is configured to.
+
+Once you submit all information, the configuration script is rendered. 
+
+## 4.2 Deploying
+
+To find the configuration script for a virt-who configuration, click it's name on virt-who configurations list page. You will see the page with brief information about how to deploy the configuration script and then the script itself.
+ 
+There are multiple ways to do the deployment. Only recent version of virt-who is supported. The scripts does the check and prints warning if unsupported version is detected. Note for Satellite 6 users, the supported version is always in Satellite Tools repository, make sure the host where you run the script has access to it or you have supported virt-who version available.
+
+### 4.2.1 Copy and paste
+
+You can simply copy and paste the content of the script to some file on host where you want to deploy it. Make sure the file is only readable by root, it contains sensitive information. Also make sure, you clean clipboard afterwards. Then simply run the script e.g. like this
+
+```sh
+bash /root/my_script
+```
+
+### 4.2.2 Downloading the script
+
+You can download the script using "Download the script" button. If you're running this on some workstation, make sure you store it on secured location and transfer it to target host by secure transport layer (e.g. ssh). The script contains sensitive information. After you copy it to secure location on target, run the script. You might need to make it executable first. Following commands illustrate the process.
```

Comment:
If you're running this on some workstation - I think it's irrelevant, you *always* store it on a secure location :-)